plasmapy/transport/collisions.py

```diff
@@ -1794,3 +1799,104 @@ def coupling_parameter(T,
                              "Something went horribly wrong.")
     coupling = coulombEnergy / kineticEnergy
     return coupling.to(u.dimensionless_unscaled)
+
+
+@particle_input
+@check_quantity()
+def Dreicer_electric_field(T: u.K,
+                           n_i: 1/u.m**3,
+                           ion: Particle,
+                           coulomb_log: numbers.Real = None,
+                           vTh: u.m / u.s = np.nan * u.m / u.s,
+                           coulomb_log_method: str = "classical",
+                           ) -> u.V / u.m:
+    r"""
```

Comment:
Why's this a raw string?

Reply:
For latex formulas in the currently empty notes section - thanks for making me check those!